proto/ai/traceable/agent/config/v1/config.proto

```diff
@@ -96,4 +96,22 @@ message SamplingConfig {
 message Javaagent {
   // set this flag to export certificates configured in JKS to libtraceable for making HTTPS connection to TPA.
   google.protobuf.BoolValue import_jks_certs = 1;
+}
+
+message LogConfig {
+  // logging mode. Allowed values are none, stdout, file.
+  google.protobuf.StringValue mode = 1;
+  // logging level. Allowed values are trace, debug, info, warn, error, critical.
+  google.protobuf.StringValue level = 2;
```

Comment:
@kaushal02 can we use enum instead of strings?

Reply:
didn't realise that was an option. Updating it now, thanks :)